source/playbooks/running-playbooks.rst

```diff
@@ -1,41 +1,30 @@
+ 
+You can also use slash commands to initiate playbook runs, make announcements, and get information about a playbook run.
 
-To run a playbook use the ``/playbook start`` slash command from any channel. You can also use the desktop or browser to:
+Each playbook has a dashboard with details such as how many runs using the playbook are currently in progress, total runs per week over a 12-week period, and a list of runs with details such as status, duration, owner/participants, and tasks finished.
 
- * Select the **Playbook** icon in the channel header, and select **Run playbook** from the right-hand sidebar.
- * Use the context menu of a post and select **Run playbook**.
+.. image:: ../images/Playbook-dashboard.png
+   :alt: View and track playbook information.
 
-Running a playbook opens an interactive dialog. You can create a new playbook or select an existing playbook. Provide a name for your playbook run, then select **Start run**. Only playbooks you belong to are listed in the **Playbook** drop-down menu. The creator of a playbook run is automatically added as the first member and becomes the owner. Lastly, when the playbook run is created, it's announced in the associated channel by the Playbook Bot.
+Join a playbook run
+-------------------
 
-Joining a playbook run
-~~~~~~~~~~~~~~~~~~~~~~
+Open Playbooks to see a list of runs, to which you have access. If the channel is public, you can join it without permission. You can also search for and join the channel via **Find channel** in the channel sidebar.
 
-Open **Main Menu > Playbooks** to see a list of runs, to which you have access. If the channel is public, you may join the channel without permission. You can also search for and join the channel via **Browse Channels**.
+If the run channel is private, an existing member must invite you.
 
-If the channel is private, an existing member must invite you.
+Work with checklists
+~~~~~~~~~~~~~~~~~~~~
 
-Working with tasks
-~~~~~~~~~~~~~~~~~~
+Checklists contain tasks - they can be pre-configured task templates and they can also be added, edited, and removed as needed during an active run. Any member of the playbook run can work with tasks. Note that if you edit, add, or delete a task during an active run it applies only to that run. The change isn't applied to the playbook. If you decide to retain that change, edit the playbook's checklist.
 
-Tasks can be part of pre-configured task templates in playbooks and they can also be added, edited, and removed as needed during an active run. Any member of the playbook run can work with tasks:
+If you aren't able to open the checklists to see completed tasks, hover over any area of the checklist. Then in the top right corner, select the filter icon. Under the **Task State** section, select **Show checked tasks**.
```

Comment:
@itao, we should prioritize a solution here, as it's awkward to have to add this to documentation to bring clarity.